server/src/test/java/com/cloud/vm/UserVmManagerImplTest.java

```diff
@@ -245,6 +249,45 @@ public class UserVmManagerImplTest {
     @Mock
     ServiceOfferingJoinDao serviceOfferingJoinDao;
 
+//    @Mock
```

Comment:
I see these are commented, can you please delete them

Reply:
Fixed.